modules/tinymce/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
 - New `color_map_background` and `color_map_foreground` option which sets the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
-- Added optional `storageKey` property to colorinput component #TINY-9184
+- Added optional `storageKey` property to colorinput component and color swatch-type fancy menu items #TINY-9184
```

Comment:
```suggestion
- Added optional `storageKey` property to `colorinput` component and `colorswatch` fancy menu items. #TINY-9184
```

They are actual names so should have backticks. You also need to fix the bridge changelog as well.

- [ ] Fix bridge changelog as well